contracts/wawaka/contract-build.cmake

```diff
@@ -80,11 +51,7 @@ LIST(APPEND WASM_BUILD_OPTIONS "-std=c++11")
 LIST(APPEND WASM_BUILD_OPTIONS "-DUSE_WASI_SDK=1")
 
 SET(WASM_LINK_OPTIONS)
-LIST(APPEND WASM_LINK_OPTIONS "-Wl,--initial-memory=${LINEAR_MEMORY}")
-LIST(APPEND WASM_LINK_OPTIONS "-Wl,--max-memory=${LINEAR_MEMORY}")
-LIST(APPEND WASM_LINK_OPTIONS "-z stack-size=${INTERNAL_STACK_SIZE}")
 LIST(APPEND WASM_LINK_OPTIONS "-Wl,--allow-undefined")
```

Comment:
These are build/link-time memory options, hence under the control of developers/users. I am wondering if any of these should be checked (somehow) by the eservice before running the related contract for safety or efficiency reasons.
The interpreter's memory configuration should define the boundaries of a contract's memory configuration, so safety should not be an issue. However, contracts could still always max out memory. Does that determine an efficiency issue for the eservice or an unfair allocation w.r.t. other contracts?

Reply:
i think you are missing... these are build time options for the CONTRACTS not for the interpreter. anyone building a contract can easily add these specifications if they want. we do not require them. as it stands... if the numbers are too big then the interpreter will throw the error.